src/components/FacebookFeed.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+export class FacebookFeed extends React.Component<any, any> {
+    constructor(props: any) {
+        super(props)
+        this.state = { width: 0, height: 0 }
+        this.updateWindowDimensions = this.updateWindowDimensions.bind(this)
+    }
+
+    componentDidMount() {
+        this.updateWindowDimensions()
+        window.addEventListener('resize', this.updateWindowDimensions)
+    }
+
+    componentWillUnmount() {
+        window.removeEventListener('resize', this.updateWindowDimensions)
+    }
+
+    updateWindowDimensions() {
+        this.setState({ width: window.innerWidth, height: window.innerHeight })
```

Comment:
`Component.setState()` should always accept an arrow function as an argument. This is because `setState()` is not garanteed to be synchronous (although it often is) and can cause problems when using the previous state value to set the new one (`setState((previousState: S) => S)`).

```suggestion
        this.setState(() => ({ width: window.innerWidth, height: window.innerHeight }))
```

Reply:
Oh I didn't knew that, thanks ! I guess thats why frameworks like Redux exists.. ?

Reply:
Exactly! Redux lets you manage stores and handle state-changing actions efficiently:)